### DISCUSSION TITLE
Is there a godoc/javadoc/etc equivalent in cue for documenting cue files

### DISCUSSION THREAD

**Discussion by jugaadi** [cuelang/cue#997] [Q&A] [answered]
The motivation here is to produce a structured documentation for cue files. Something equivalent to JSON Schema's`title` and `description` annotations would be really helpful.

**Comment by myitcv** [comment, marked as answer]
This has been discussed many times in passing, but I've now created an issue to which we can refer people: https://github.com/cuelang/cue/issues/998

In short; `cue doc` will look and feel very much like `go doc`.

**Comment by cueckoo** [comment]
This discussion has been migrated to https://github.com/cue-lang/cue/issues/997.

For more details about CUE's migration to a new home, please see https://github.com/cue-lang/cue/issues/1078.
